framework/web/Request.php

```diff
@@ -1444,8 +1419,8 @@ public function validateCsrfToken($token = null)
     /**
      * Validates CSRF token
      *
-     * @param string $token
-     * @param string $trueToken
+     * @param string $token The masked client-supplied token.
```

Comment:
> this part of the code hasn't changed during this review cycle.
> At some point it starts to feel like hitting a moving target.

This is understandable and not comfortable. On the other hand, like a PR itself, reviewing isnt always done 100% right the first time. Practice shows that a fresh relook is likely to show some new areas for improvement. Therefore it is something that can be expected.

Its up to you, if you share the ideas to improve this PR, I would prefer this to include it and not postpone, as postponing leads to a timeconsuming future.